src/Tribe/Service_Providers/Context.php

```diff
@@ -44,32 +45,33 @@ public function register() {
 	 * @return array The modified context locations.
 	 */
 	public function filter_context_locations( array $locations = [] ) {
-		$locations = array_merge(
+		$view_query_var = View::query_var();
+		$locations      = array_merge(
 			$locations,
 			[
-				'event_display'        => [
+				'event_display'      => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 8 space(s) between "'event_display'" and double arrow, but found 6.

<sub>hash: fe8f3c23de919ad0fba0b8d59c96b146</sub>